locksmith/src/routes/user.js

```diff
@@ -13,5 +13,9 @@ router.get(
   userController.retrieveRecoveryPhrase
 )
 router.put('/:emailAddress', userController.updateUser)
+router.put(
+  '/:emailAddress/payment_details',
```

Comment:
I think the difference wuth the previous route is that this one only happens when the user is logged in?

Reply:
There is more of an operational difference around not modifying the working end point while we worth through rolling this out. The service layer doesn't exactly have a concept of a logged in user, at the time of writing the caller of the API determines when this will be called. 

With a valid stripe token, the users details will be updated.  When comfortable with the code we can merge this into the standard user updated endpoint if that seems like the right path

Reply:
nit do you mind using `paymentdetails` since we use `recoveryphrase` or `privatekey`?